config/modules.php

```diff
@@ -89,7 +93,7 @@
             'filter'           => ['path' => 'Http/Middleware', 'generate' => true],
             'request'          => ['path' => 'Http/Requests', 'generate' => true],
             'routes'           => ['path' => 'Http/Routes', 'generate' => true],
-            'provider'         => ['path' => 'Providers', 'generate' => true],
+            'provider'         => ['path' => 'Providers', 'generate' => false],
```

Comment:
What happens now then when a new module is generated, where does the provider come from?

Reply:
Yeah, now the provider comes from resources/views/stubs folder.